keras2onnx/ke2onnx/simplernn.py

```diff
@@ -77,6 +83,11 @@ def convert_keras_simple_rnn(scope, operator, container):
     rnn_output_names.append(rnn_y_name)
     rnn_output_names.append(rnn_h_name)
     oopb = OnnxOperatorBuilder(container, scope)
+
+    if uses_masking_layer:
+        mask_cast = oopb.add_node('Cast', operator.input_masks[0].full_name, operator.full_name + '_mask_cast', to=oopb.int32)
```

Comment:
Can you replace add_node with oopb.apply_cast(...)? I know you probably was mislead by some bad examples that we haven't fixed yet. but with apply_cast function set, the converter will be transparent with the different opset.

Reply:
I've made the replacement.